packages/flutter/lib/src/material/dialog.dart

```diff
@@ -49,6 +49,7 @@ class Dialog extends StatelessWidget {
     this.insetAnimationDuration = const Duration(milliseconds: 100),
     this.insetAnimationCurve = Curves.decelerate,
     this.shape,
+    this.clipBehavior = Clip.none,
     this.child,
   }) : super(key: key);
```

Comment:
Since you say in the comments that it must not be null, should you add an `assert(clipBehavior != null)` ?

Reply:
Good call, done!